### PR TITLE
issue template update

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Update the issue
-
-

Review please

### CODE REVIEW

**Review by Nightenom — CHANGES_REQUESTED** [review]
also the second and the third one :D

**Comment by Raycoms** [comment]
I would delete #3, I will update #2

File: .github/ISSUE_TEMPLATE/10-bug-report.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      description: Please check to see if any newer version fixes the issue you're experiencing.
+      options:
+        - label:  I am running the latest **alpha** version of MineColonies for my Minecraft version.
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Did you check on the Wiki? or ask on Discord?
+      description: Please check on our wiki (https://wiki.minecolonies.ldtteam.com/) or ask on discord (https://discord.minecolonies.com/) if your issue is covered there.
+      options:
+        - label: I checked the MineColonie Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.
```
Comment (uecasm) [inline .github/ISSUE_TEMPLATE/10-bug-report.yaml]: still

Comment (Raycoms) [reply .github/ISSUE_TEMPLATE/10-bug-report.yaml]: What here?

Comment (uecasm) [reply .github/ISSUE_TEMPLATE/10-bug-report.yaml]: "Minecolonie", like I mentioned on discord

**Pull request merged**